app/src/main/res/layout/activity_leaderboard_listitem.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm not totally familiar with xml yet, so reviewing this is a little hard, but it seems fine.